v2/manifests/core-ons/dp-frontend-feedback-controller.yml

```diff
@@ -0,0 +1,26 @@
+services:
+  dp-frontend-feedback-controller:
+    x-repo-url: "https://github.com/ONSdigital/dp-frontend-feedback-controller"
+    build:
+      context: ${DP_REPO_DIR:-../../../..}/dp-frontend-feedback-controller
+      dockerfile: Dockerfile.local
+    command:
+      - reflex
+      - -d
+      - none
+      - -c
+      - ./reflex
+    volumes:
+      - ${DP_REPO_DIR:-../../../..}/dp-frontend-feedback-controller:/dp-frontend-feedback-controller
+    expose:
+      - "25200"
+    ports:
+      - 25200:25200
+    restart: unless-stopped
+    environment:
+      BIND_ADDR:                          ":25200"
+    healthcheck:
+      test: ["CMD", "curl", "-sSf", "http://localhost:25200/health"]
+      interval: ${HEALTHCHECK_INTERVAL:-30s}
+      timeout: 10s
+      retries: 10
```

Comment:
New line at EOF please - you can get an extension to do this on save.

Reply:
Done